mscore/menus.cpp

```diff
@@ -1414,6 +1414,7 @@ PalettePanel* MuseScore::newLinesPalettePanel()
       ottava = new Ottava(gscore);
       ottava->setOttavaType(OttavaType::OTTAVA_22MB);
       ottava->setLen(w);
+      ottava->setPlacement(Placement::BELOW);
```

Comment:
May not be needed, certainily not for thwe issue at hane but found this to be pretty inconsistent, at least.